adapters/ba.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/home/randers/Downloads/bash-0.3.2/out/bin/bash
```

Reply:
no

Reply:
I will report you to the hacktoberfest police 😡